### PR TITLE
tiup: add documents of publish, grant, and genkey

### PR DESCRIPTION
**Pull request by CharLotteiu**
### What is changed, added or deleted? (Required)

Add three TiUP reference docs.

- tiup/tiup-command-mirror-publish.md
- tiup/tiup-command-mirror-clone.md
- tiup/tiup-command-mirror-genkey.md

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/4868
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-command-mirror-grant.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: tiup mirror grant
+---
+
+# tiup mirror grant
+
+The `tiup mirror grant` command is used to introduce a component owner to the current mirror.
+
+Component owners can use their keys to publish new components or to modify components they previously published. Before adding a new component owner, current component owner needs to send his or her own public key to the mirror administrator.
```
Comment (lucklove) [inline tiup/tiup-command-mirror-grant.md:9]:
```suggestion
Component owners can use their keys to publish new components or to modify components they previously published. Before adding a new component owner, current component owner being adding needs to send his or her own public key to the mirror administrator.
```

File: tiup/tiup-command-mirror-grant.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+```
+
+`<id>` stands for the component owner's ID, which must be unique in the whole mirror. It is recommended to use an ID that matches the regular expression `^[a-z\d](? :[a-z\d]|-(? =[a-z\d])){0,38}$`.
+
+## Options
+
+### -k, --key
+
+- Specifies the key of the introduced component owner. This key can either be public or private. If it is a private key, TiUP converts it to the corresponding public key before stores it in the mirror.
+- A key can only be used by a component owner.
```
Comment (lucklove) [inline tiup/tiup-command-mirror-grant.md:28]:
```suggestion
- A key can only be used by just one component owner.
```

**Review by lucklove — COMMENTED** [review]
Rest LGTM

File: tiup/tiup-command-mirror-grant.md (hunk 1 above)
Comment (CharLotteiu) [reply tiup/tiup-command-mirror-grant.md:9]:
```suggestion
Component owners can use their keys to publish new components or to modify components they previously published. Before adding a new component owner, the current component owner being added needs to send his or her own public key to the mirror administrator.
```

File: tiup/tiup-command-mirror-grant.md (hunk 2 above)
Comment (CharLotteiu) [reply tiup/tiup-command-mirror-grant.md:28]:
```suggestion
- A key can be used by only one component owner.
```

**Review by TomShawn — COMMENTED** [review]
minor comments

**Comment by Joyinqin** [comment]
@TomShawn PTAL again, thx~

File: tiup/tiup-command-mirror-grant.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: tiup mirror grant
+---
+
+# tiup mirror grant
+
+The `tiup mirror grant` command is used to introduce a component owner to the current mirror.
+
+Component owners can use their keys to publish new components or to modify components they previously published. Before adding a new component owner, the current component owner being added needs to send his or her own public key to the mirror administrator.
```
[the remark this answers is not in the document]
Comment (lucklove) [reply tiup/tiup-command-mirror-grant.md:9]: Correct.

**Pull request merged**